<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeManager</class>
 <widget class="QMainWindow" name="CodeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>891</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="appLabel">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Code Manager</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="devLabel">
         <property name="text">
          <string>Developer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="devComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="activitiesGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Activities</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>751</width>
          <height>227</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTreeView" name="activitiesView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="activitiesButtonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
           <property name="centerButtons">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="changesetsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Changesets</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>751</width>
          <height>227</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeView" name="changesetsView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="changesetsButtonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="detailsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Details</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>751</width>
          <height>153</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="releasesTab">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="title">
             <string>Files</string>
            </attribute>
            <widget class="QWidget" name="verticalLayoutWidget_4">
             <property name="geometry">
              <rect>
               <x>-1</x>
               <y>-1</y>
               <width>761</width>
               <height>191</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
               <widget class="QTreeView" name="releasesView">
                <property name="itemsExpandable">
                 <bool>false</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="regressionsTab">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="title">
             <string>Regressions</string>
            </attribute>
            <widget class="QWidget" name="verticalLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>761</width>
               <height>191</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QTreeView" name="regressionsView"/>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="improvementsTab">
            <attribute name="title">
             <string>Improvements</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="detailsButtonBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
    <zorder>changesetsGroupBox</zorder>
    <zorder>activitiesGroupBox</zorder>
    <zorder>detailsGroupBox</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
